src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java

```diff
@@ -311,6 +311,13 @@ public boolean isPaymentPlanRejected() {
             && caseData.hasApplicantRejectedRepaymentPlan());
     }
 
+    @Override
+    public boolean isHwFClaimSubmit() {
+        return caseData.isHWFOutcomeReady()
+            && caseData.isHWFTypeClaimIssued()
+            && caseData.getCcdState().equals(CaseState.PENDING_CASE_ISSUED);
+    }
```

Comment:
Do you need caseData.isHWFOutcomeReady() here? Check implementation of isHWFOutcomeReady(), I think it is doing same checks..

Use == to compare enums

Reply:
done